src/app/auto-login/page.tsx

```
@@ -0,0 +1,63 @@
+        }
+    }, [isAuthenticated, isLoading, signIn]);
+
+    if (errorMsg) {
+        return (
+            <div className="flex h-screen w-full flex-col items-center justify-center bg-[#4A0D68] text-white">
+                <p className="text-xl font-bold text-red-400">Auto-login Failed</p>
+                <p className="mt-2 text-sm">{errorMsg}</p>
+                <button 
+                    onClick={() => window.location.reload()} 
+                    className="mt-4 rounded bg-white px-4 py-2 text-[#4A0D68]"
+                >
```

Biome (2.4.13)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)